ddsketch/pb/ddsketch.proto

```diff
@@ -0,0 +1,66 @@
+/* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2020 Datadog, Inc.
+ */
+
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/sketches-go/ddsketch/pb/sketchpb";
```

Comment:
Do we want to check in a Python `pb` file and point to that instead of the Go version? I know very little about protobuf / how it works in dynamic languages

Reply:
This can be removed for python - it defines the import path of the generated code for go.